cmd/swapcli/errors.go

```diff
@@ -4,11 +4,10 @@ import (
 	"fmt"
 )
 
-//nolint:lll
 var (
-	errNoDuration                         = fmt.Errorf("must provide non-zero --duration")
-	errCannotHaveGreaterThan100Commission = fmt.Errorf("%s must be less than 1", flagRelayerCommission)
-	errMustSetRelayerEndpoint             = fmt.Errorf("%s must be set if %s is set", flagRelayerEndpoint, flagRelayerCommission)
+	errNoDuration                  = fmt.Errorf("must provide non-zero --duration")
+	errCannotHaveGreaterThan100Fee = fmt.Errorf("%s must be less than 1", flagRelayerFee)
```

Comment:
Can we make the error message, the check, and the name of the error agree?  The message says that it must be strictly less than, but the check is less than or equal to 100%.  You could call the error `errFeeNotLessThan100Percent` or `errFeeGreaterThan100Percent` depending on the check. Or just `errRelayerFeeTooHigh`.